clients/dpd-client/build.rs

```diff
@@ -92,7 +93,61 @@ fn main() -> Result<()> {
                     slog::debug!(state.log, "client response"; "result" => ?result);
                 }
             })
-	    .with_derive("PartialEq")
+            .with_derive("PartialEq")
+            // NOTE: This should all go away when we can open-source the real
+            // `dpd-client` from the repo itself. These are used to ensure that
+            // we pick up things like the display implementations.
+            .with_replacement(
+                "ActiveCableMediaInterfaceId",
+                "transceiver_decode::ActiveCableMediaInterfaceId",
+                std::iter::once(TypeSpaceImpl::Display)
```

Comment:
do you potentially want to make a `static` table and then

```
for entry in table {
    settings.with_replacement(entry);
}
```

... more or less? My logic is that it might be easier to maintain relative to changes in the `transceiver_decode` crate.

Reply:
Done in 895d2da07